ixmp/utils.py

```diff
@@ -390,10 +390,11 @@ def describe(df):
         platform.scenario_list(model=model, scen=scenario, default=default_only)
         .groupby(["model", "scenario"])
         .apply(describe)
-        .reset_index()
     )
 
-    if not len(info):
+    if len(info):
```

Comment:
I realize I'm a little late to the party and it doesn't really matter but just as a side note, you don't need the `if len(info):` just `if info:` is fine since and empty list evaluates to false (https://docs.python.org/3/library/stdtypes.html#truth-value-testing).

Reply:
True - thanks :)

Reply:
`info` is `pd.DataFrame`, not `list`:
```py
In [1]: df = pd.DataFrame()

In [2]: if df:
   ...:     print("foo")
   ...: 
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-559d78b99c9e> in <module>
----> 1 if df:
      2     print("foo")
      3 

~/.local/lib/python3.9/site-packages/pandas/core/generic.py in __nonzero__(self)
   1440     @final
   1441     def __nonzero__(self):
-> 1442         raise ValueError(
   1443             f"The truth value of a {type(self).__name__} is ambiguous. "
   1444             "Use a.empty, a.bool(), a.item(), a.any() or a.all()."

ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().

```

Reply:
A right, good point, ignore the comment then